src/me/payment/me-payment.service.js

```diff
@@ -0,0 +1,12 @@
+angular.module('ovh-api-services').service('OvhApiMePayment', function ($injector) {
+  'use strict';
+
+  return {
+    Method: function () {
+      return $injector.get('OvhApiMePayMethod');
```

Comment:
```suggestion
      return $injector.get('OvhApiMePaymentMethod');
```
?

Reply:
Good catch but I can't use `OvhApiMePaymentMethod` because some other resources already use these names. These resources manage some routes that will be removed once the migration of payment method will be finished in US side. See https://github.com/ovh-ux/ovh-api-services/tree/master/src/me/paymentMehtod